AsyncAwait.Task2.CodeReviewChallenge/Middleware/StatisticMiddleware.cs

```diff
@@ -1,42 +0,0 @@
-﻿using System;
```

Comment:
Removed this middleware because in my opinion it is not ok to block the middleware pipeline with I/O calls to third party services